README.md

```diff
@@ -134,18 +169,14 @@ The following example will generate Magento 2.2 documentation only.
      - /Rakefile
    ```
 
-1. Run the preview command:
-
-   ```
-   $ rake preview
-   ```
-   This command:
-   * Checks your environment according to the dependencies in `Gemfile.lock`.
-   * Removes the `_site/` directory, which contains previously generated preview files.
-   * Generates a new preview and opens the landing page in a web browsers.
+2. Run [`rake preview`](#using-rake)
```

Comment:
The step should say what you are doing.
"Use the following rake command to generate a preview in a web browser."